x-pack/plugins/ml/server/routes/schemas/modules.ts

```diff
@@ -107,6 +107,13 @@ export const moduleIdParamSchema = schema.object({
   moduleId: schema.string({ meta: { description: 'ID of the module' } }),
 });
 
+export const optionalSizeQuerySchema = schema.object({
+  /**
```

Comment:
@darnautov should this comment be moved into a `meta` prop for the schema now?

Reply:
yes, for example https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/server/routes/schemas/anomaly_detectors_schema.ts#L31

Reply:
Updated in 483c2bb5d94bb576142c1a37e816289b7857cacb